test/uk/gov/hmrc/agentaccesscontrol/controllers/AuthorisationControllerSpec.scala

```diff
@@ -35,11 +36,12 @@ import scala.concurrent.{ExecutionContext, Future}
 class AuthorisationControllerSpec extends UnitSpec with BeforeAndAfterEach with MockitoSugar {
 
   val fakeRequest = FakeRequest("GET", "/agent-access-control/sa-auth/agent//client/utr")
+  val payeFakeRequest = FakeRequest("GET", "/agent-access-control/paye-auth/agent/A11112222A/client/123%2F123456 ")
 
   val auditService = mock[AuditService]
   val authorisationService = mock[AuthorisationService]
   val mtdAuthorisationService = mock[MtdAuthorisationService]
-  val controller = new AuthorisationController(auditService, authorisationService, mtdAuthorisationService)
+  def controller(enabled : Boolean) = new AuthorisationController(auditService, authorisationService, mtdAuthorisationService, Configuration("features.allowPayeAccess" -> enabled))
```

Comment:
could add a default value here for the common case (enabled) so that the existing tests wouldn't need to specify payeEnabled
`def controller(enabled: Boolean = true)`